website/components/subnav/index.jsx

```diff
@@ -27,6 +27,7 @@ export default function ProductSubnav() {
       menuItems={subnavItems}
       constrainWidth
       Link={Link}
+      matchOnBasePath
```

Comment:
Still curious about this one - is there any place it's not needed?

Reply:
All the subnavs in www don't need it.